core/templates/domain/topic/TopicObjectFactorySpec.ts

```diff
@@ -109,6 +110,28 @@ describe('Topic object factory', () => {
     ]);
   });
 
+  it('should warn user when no skill Ids were added for the diagnostic test',
```

Comment:
Here and elsewhere, either IDs or ids, but not Ids.

Reply:
Updated